src/main/resources/js/lgtm-addon.js

```diff
@@ -78,69 +79,68 @@ function loadProjects() {
 }
 
 function handleProjectChange(event) {
-	clearSelect2Field(issueTypeFieldId);
+	lgtmIssueTypeId = null;
 	clearSelect2Field(priorityFieldId);
 	clearSelect2Field(closedStatusFieldId);
 	clearSelect2Field(reopenedStatusFieldId);
 	
 	if (AJS.$(projectFieldId).select2('val') !== "none") {
-		loadIssueTypes(AJS.$(projectFieldId).select2('val'));
+		loadIssueType(AJS.$(projectFieldId).select2('val'));
 		loadPriorities(AJS.$(projectFieldId).select2('val'));
 	}
 }
 
-function loadIssueTypes(projectKey) {
+function loadIssueType(projectKey) {
 	$.ajax({
 		url : AJS.contextPath() + "/rest/api/2/project/" + projectKey + "/statuses"
 	}).done(
 	function(issueTypes) {
-		var fieldContent = issueTypes.map(function(issueType) {
-			return {
-				value : issueType.id,
-				text : issueType.name
-			}
+		var lgtmIssueType = issueTypes.find(function (element) {
+			return element.name === "LGTM Alert";
 		});
-		fieldContent.unshift({value : "none", text : ""});
 		
-		renderSelect2Field(issueTypeFieldId, fieldContent);
-		AJS.$(issueTypeFieldId).on('change', function(e) {
-			handleIssueTypeChange(issueTypes);
-		});
-	
-		if (config !== null) {
-			changeSelect2Value(issueTypeFieldId, config.issueTypeId);
+		if (lgtmIssueType === undefined) {
+			AJS.messages.error("#message-context", {
+				title : 'The project ' + projectKey + ' uses an issue type scheme that does not contain the "LGTM alert" issue type.', 
+				closeable : true,
+				fadeout : false
+			});
+			
+			var messageContent = document.createElement('p');
+			messageContent.textContent = 'Please go to the project configuration and add it:';
+			
+			var link = document.createElement('a');
+			var configUrl = AJS.params.baseURL + '/plugins/servlet/project-config/' + projectKey + '/issuetypes';
```

Comment:
I think the URL is indeed the best page to direct  the user to. However, it is not entirely clear what to do on the page. I think something like this would be clearer:
```
The "LGTM alert" issue type is not included in the issue type scheme of PROJECT_NAME.
It can be added using the "Edit issue types" action in PROJECT_NAME's <a href="LINK_URL">issue type settings</a>  
```

Reply:
Also we should probably open that link in a new tab/pag . That is `target=_blank` or so, right?